NAMESPACE

```diff
@@ -1,9 +1,15 @@
 # Generated by roxygen2: do not edit by hand
 
 export(create_visc_project)
```

Comment:
When I rebuild I get a `export(check_pandoc_version)` line added to the namespace.

Reply:
Forgot to commit this change! It's updated now.